common/djangoapps/cme_registration/tests/registration_tests.py

```diff
@@ -405,8 +404,7 @@ def test_db_records_with_others_created(self):
         self.post_vars['specialty_free'] = 'Patient care'
         self.post_vars['sub_specialty'] = 'Other'
         self.post_vars['sub_specialty_free'] = 'Legs and feet'
-        self.post_vars['affiliation'] = 'Other'
-        self.post_vars['other_affiliation'] = 'other_affiliation'
+        self.post_vars['affiliation'] = 'Not affiliated with Stanford Medicine'
```

Comment:
Wait, shouldn't we delete this entire test case?
`with_others_created` seems to imply that this is related to users selecting "other; isn't that no longer a valid option?